docs/quick_start/index.md

```diff
@@ -0,0 +1,118 @@
+- **Seq**: Log system designed for searching, analysis, and visualization of structured log data.
+- **Jaeger**: An open-source tool for tracing requests.
+- **Traefik**: A cloud-native application proxy that routes incoming requests to the appropriate microservices.
+- **OpenTelemetry**: A set of APIs, SDKs, and tools to generate, collect, and export telemetry data.
+- **Metabase**: Tool for visualizing and sharing data insights.
+
+To configure the local cluster, start by navigating to the `dev-cluster` directory. Once there, ensure that you've set the `sendgrid_api_key` in the `terraform.tfvars` file. Without this api key, the functionality for sending emails won't be available:
+
+```terraform
+sendgrid_api_key = "my_sendgrid_api_key"
+```
+
+Then, execute the following script to set up the local cluster:
+
+```sh
+./deploy.sh
+```
+
+## Setting up the application
+
+After successfully setting up the local cluster, your next step is to initiate your application. Begin by navigating to the `backend` directory. This directory is the heart of your application, containing various subdirectories and files, each serving a specific purpose:
+
+```txt
+backend
+├── dev - Configuration for local development.
+├── release - Dockerfiles for production deployment.
+├── src
+│   ├── Apps
+│   │   ├── ProjectName.Api - Hosts the API, serving as the backend entrypoint.
+│   │   ├── ProjectName.LeanPipeFunnel - Manages real-time notifications.
+│   │   └── ProjectName.Migrations - Handles database migrations.
+│   └── ContextName
+│       ├── ProjectName.ContextName.Contracts - API client interfaces.
+│       ├── ProjectName.ContextName.Domain - Core business logic.
+│       └── ProjectName.ContextName.Services - Service layer.
+├── tests
+│   ├── ContextName
+│   │   ├── ProjectName.ContextName.Domain.Tests - Tests for business logic.
+│   │   └── ProjectName.ContextName.Services.Tests - Tests for service layer.
+│   ├── ProjectName.IntegrationTests - Comprehensive application-wide tests.
+```
+
+### Generating initial migrations
+
+Let's start with generating initial migrations. Begin by navigating, to the `backend/src/Apps/ProjectName.Migrations` directory. Generate initial migrations using the `dotnet ef` tool (install it via `dotnet tool install --global dotnet-ef`):
+
+```sh
+# It does not need to point to a real database
+export PostgreSQL__ConnectionString='Host=localhost;Database=app;Username=app;Password=Passw12#'
+dotnet ef migrations add --context ContextNameDbContext -o Migrations InitialMigration
+```
+
+### Starting the application
+
+Once initial migrations have been generated we can start the application. First of all ensure that you've selected the `kubectl` context corresponding to the newly created cluster:
+
+```sh
+kubectl config use-context k3d-projectname
+```
+
+Then you can run the migrations and the API using the following command:
+
+```sh
+tilt up migrations api
```

Comment:
This will run API only, it will not start migrations.

Reply:
This is what we always had in `README` if I remember correctly it did start the migrations when I ran it. Anyway I separated those commands now.